QRCodeReaderViewController/QRCodeReader.m

```diff
@@ -153,14 +153,18 @@ - (void)toggleTorch
 - (void)startScanning
 {
   if (![self.session isRunning]) {
-    [self.session startRunning];
+      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_HIGH, 0), ^{
```

Comment:
@oa-s I didn't check further, but this is safe to run in the background?

Reply:
tested with a real devide, was good.